Readme.md

```diff
@@ -73,23 +78,21 @@ This is a [Node.js](https://nodejs.org/en/) module available through the
 npm install bytes-iec
 ```
 
-
 ## Usage
 
 ```js
-var bytes = require('bytes');
+var bytes = require('bytes-iec');
 ```
 
 #### bytes.format(number value, [options]): string|null
 
-Format the given value in bytes into a string. If the value is negative, it is kept as such. If it is a float, it is
- rounded.
+Format the given value in bytes into a string. If the value is negative, it is kept as such. If it is a float, it is rounded.
 
 It supports the following output formats:
 
- * `binary`: uses the binary prefixes (KiB, MiB...)
- * `decimal`|`metric`: uses the metric system (decimal) prefixes (kB, MB...)
- * `jedec`|`compatibility`: uses the binary units, but the metric prefixes (kB == 1024B, MB...)
+ * `binary`: uses the binary units (KiB, MiB, ...)
```

Comment:
considering all the functions use the same MODE info, perhaps this should be its own section…  And all of them can link back to the Modes

Reply:
I think, since there are already sections about the 3 main modes, perhaps each section should get a list of its supported synonyms/codes next to its details, and the parameter listings link to the main heading?

Reply:
Working on this actually made a question arise for me: how does the API respond to mixing the `unit` and `mode` arguments? It seems the `mode` parameter is just ignored when the `unit` is incompatible
![image](https://user-images.githubusercontent.com/3950300/101243256-2f1e3000-36ff-11eb-97cb-13743fa386e9.png)

Reply:
I feel we should add a hint that `mode` really only affects the interpretation of metric units. The binary units are unambiguous in every situation.

Reply:
well it should affect format when you don't specify a unit.

Reply:
And parse is unaffected except for compatibility mode

Reply:
Right, so it both sets a type of units to favor when unspecified, and determines the value of metric units when present

Reply:
That sounds very concise!  perfect to put in the mode section

Reply:
I've added section links so people can click-through, and tried to clarify all the terminology: metric units can be decimal/base-10 (SI) or binary/base-2 (jedec, legacy, compatibility)

Reply:
Oh, this whole section as part of bytes.format is redundant now actually, better only refer to the centralized documentation

Reply:
I wouldn't say so.  Having a listing right in the "Modes" section, linking to each mode seems very useful.  Quick reference for the choices.

Reply:
Done

Reply:
Forgot to update the links for `mode` in the API, pushing in a minute